src/sdsc/xslt.py

```diff
@@ -0,0 +1,396 @@
+#
```

Comment:
I think I'd like it much better if all the extension functions got an own file (while we are at it; however, stuff like dupecheck and dupecheckmessage should still be in the same file). Because really, what you are doing here is just taking 2/3 of the existing code and putting it all in a single file. Also, if the Python code ever grows, I predict, it is largely going to be growing more extension functions.